lib/create-custom-runner.ts

```diff
@@ -16,10 +41,11 @@ const createRemoteCache = (
     implementation,
     options
   );
+  const [readDisabled, writeDisabled] = getReadWriteDisabledStatus(options);
```

Comment:
I personally dislike negative options.
"Have you disabled the disable writes option ?" always sounds weird to me.